src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewController.swift

```diff
@@ -262,11 +263,12 @@ extension ExposureDetectionViewController {
 // MARK: - CountdownTimerDelegate methods.
 
 extension ExposureDetectionViewController: CountdownTimerDelegate {
-	func update(time: String) {
+
+	func countdownTimer(_ timer: CountdownTimer, didUpdate time: String) {
```

Comment:
why are you passing the timer, if you are not using it? is it necessary?

Reply:
This is a kinda pattern of delegate method. It's always good to pass the "event source" or "sender".

Reply:
Hm. But isn't this against YAGNI principle?

Reply:
It is roughly following the guidelines by apple:
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html

I agree it is not necessarily required, however I would favour to follow the convention and pass the reference as first parameter.

Reply:
Thank you for clarifying. I agree with you.